src/sage/combinat/free_module.py

```diff
@@ -151,7 +151,7 @@ class CombinatorialFreeModule(UniqueRepresentation, Module, IndexedGenerators):
     The constructed module is in the category of modules with basis
     over the base ring::
 
-        sage: CombinatorialFreeModule(QQ, Partitions()).category()
+        sage: CombinatorialFreeModule(QQ, Partitions()).category()                      # optional - sage.combinat
```

Comment:
again optional tag `sage.combinat` in a file of the combinat module. Top of the file ?

Reply:
No, this one definitely should not become a file-level annotation.
Nothing from (the higher level bits of) `sage.combinat` is used in the implementation of `CombinatorialFreeModule`, and it will be shipped (as all other free module implementations) in a distribution sagemath-modules; see https://deploy-preview-35749--sagemath-tobias.netlify.app/reference/misc/sage/features/sagemath.html#sage.features.sagemath.sage__modules and https://github.com/sagemath/sage/blob/7992ac088e1f73ca3dd4db9fc6560a704cd5034b/pkgs/sagemath-modules/MANIFEST.in.m4#L60
It is only the use of `Partitions` again that makes this annotation here necessary. In fact, if it turns out that this file does not have enough test coverage without all the combinatorial tests, then we will actually want to add more tests that are more elementary.